website/docs/hms.md

```diff
@@ -114,6 +114,17 @@ A Hudi table can directly be synced to the Hive Metastore using Hive Sync Tool
 and subsequently be queried by different query engines. For more information on the Hive Sync Tool, check 
 [Hudi Hive Metastore](https://hudi.apache.org/docs/syncing_metastore) docs.
 
+```shell md title="shell with HMS example"
+cd $HUDI_HOME/hudi-sync/hudi-hive-sync ; \
+./run_sync_tool.sh  \
+--metastore-uris '<thrift_url>' \
+--partitioned-by <partition_field> \
+--base-path '<path/to/s3/synced/hudi/table>' \
+--database <database_name> \
+--table <tableName> \
+--sync-mode hms 
+```
+
 ```shell md title="shell"
```

Comment:
nit:// Let's call with "shell with JDBC" to be specific.

Reply:
fixed.